pandas/core/window/__init__.py

```diff
@@ -1,3 +1,3 @@
-from pandas.core.window.ewm import EWM  # noqa:F401
+from pandas.core.window.exponentialmoving import ExponentialMovingWindow  # noqa:F401
```

Comment:
need `isort` to sort imports, aka, this needs to be placed under `from pandas.core.window.expanding import Expanding, ExpandingGroupby  # noqa:F401`
so looks something like below overall:
```python
from pandas.core.window.expanding import Expanding, ExpandingGroupby  # noqa:F401
from pandas.core.window.exponentialmoving import ExponentialMovingWindow  # noqa:F401
from pandas.core.window.rolling import Rolling, RollingGroupby, Window  # noqa:F401
```

Reply:
alright will get that sorted

Reply:
it's weird that the first issue 'pandas.core.window.ewm not found error' occured because in theory when I refactored the file name all the usages were renamed in pycharm. anyway changed the name back so it should be fine.

Reply:
anyways I'll leave these checks to run and check in on them in the morning they take ages.